client/my-sites/checkout/concierge-session-nudge/index.jsx

```diff
@@ -145,82 +145,77 @@ export class ConciergeSessionNudge extends React.Component {
 
 						<p>{ translate( 'What our team of experts can help you with:' ) }</p>
 
-						<p>
-							<ul className="concierge-session-nudge__checklist">
-								<li className="concierge-session-nudge__checklist-item">
-									<Gridicon
-										icon="checkmark"
-										className="concierge-session-nudge__checklist-item-icon"
-									/>
-									<span className="concierge-session-nudge__checklist-item-text">
-										{ translate( '{{b}}Design:{{/b}} Which template to choose.', {
+						<ul className="concierge-session-nudge__checklist">
+							<li className="concierge-session-nudge__checklist-item">
+								<Gridicon
+									icon="checkmark"
+									className="concierge-session-nudge__checklist-item-icon"
+								/>
+								<span className="concierge-session-nudge__checklist-item-text">
+									{ translate( '{{b}}Design:{{/b}} Which template to choose.', {
+										components: { b: <b /> },
+										comment: "This a benefit on a 'Purchase a call with us' page",
```

Comment:
The translator comment needs a minor grammar fix - `This is a benefit listed on a 'Purchase a call with us' page`. It would be nice to get this in along with this PR.